spritz/spritz-android/build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 26
+    buildToolsVersion "26.0.1"
+
+    defaultConfig {
+        minSdkVersion 14
+        targetSdkVersion 26
+        versionCode 1
+        versionName "1.0"
+    }
+}
+
+dependencies {
+    compile 'com.android.support:appcompat-v7:26.1.0'
```

Comment:
I think both the `support` dependencies here are unused

Reply:
Done in f44b30d478853854fb618a3dbc9881bed4d37c42.